src/entities/Criteria.java

```diff
@@ -0,0 +1,41 @@
+package src.entities;
+
+public class Criteria {
+    /**
+     * Evaluates password against pre-defined criteria
+     * If password fails a criteria, suggestion is given to improve it
+     * Invoking function can check if the suggestion is empty, then password is strong
+     *
+     * @param password
+     * @return Suggestion
+     */
+    public Suggestion evaluatePassword(String password) {
+        Suggestion suggestion = new Suggestion();
```

Comment:
The if/else sequence below means that only one suggestion will ever be added. Should the if statements be separated so that more than one violation can be found with a single call on evaluatePassword?